benchmark/README.md

```diff
@@ -141,23 +271,89 @@ To add a new multiple file test:
     `SWIFT_MULTISOURCE_SWIFT3_BENCHES`, and set `YourTestName_sources` to the
     list of source file paths.
 
-3.  Edit `main.swift`. Import and register your new Swift module.
+3.  Edit `main.swift`. Import and register your new Swift module as with the
+    single-source files.
+
+**NOTE:** we do not need to update the swiftpm build since our Package.swift will
+dynamically create targets based off the file's location. So nothing is needed
+as long as we keep our source layout the same.
+
+### The number of iterations per sample
+
+A commonly confused concept when using the low level ``Benchmark_Driver`` is the
+distinction in between iteration and sample. All benchmarks in the test suite
+take a value of N that is used to ensure that when we run an individual test
+sample the test runs are significantly longer than system jitter so we can
+distinguish in between system jitter and normal cpu performance.
+
+This is done by the low level benchmark driver will measure the time taken for
```

Comment:
"This is done by the low level benchmark driver. It will measure the time taken for"